ui/admin/tests/acceptance/auth-methods/selectors.js

```diff
@@ -90,3 +90,10 @@ export const TABLE_FIRST_ROW_ACTION_DROPDOWN =
   'tbody tr:first-child td:last-child button';
 export const TABLE_FIRST_ROW_ACTION_FIRST_ITEM =
   'tbody tr:first-child td:last-child ul li:first-child button';
+
+export const SEARCH_BAR_INPUT = '[type=search]';
+
+export const TABLE_ACTION_DROPDOWN = (authMethodId) =>
+  `tbody [data-test-auth-methods-table-row="${authMethodId}"] .hds-table__td:last-child .hds-dropdown button`;
+export const TABLE_ACTION_DROPDOWN_MAKE_PRIMARY = (authMethodId) =>
```

Comment:
With this one, I decided to get rid of the data-test selector `data-test-make-primary-action`.

Since we are already using a data-test-selector for each authMethod within the table list I found I could leverage that to select the Make primary option without using a data test selector.

Open to revert this approach if the data-test selector is preferred.

Reply:
I'm ok with this approach, makes sense to utilize the authMethod's data-test-selector